src/main/java/mekanism/common/block/BlockBounding.java

```diff
@@ -115,18 +128,6 @@ public float getPlayerRelativeBlockHardness(IBlockState state, EntityPlayer play
 		}
 	}
 
-	@Override
```

Comment:
Why

Reply:
This is not needed because ```getDrops``` is overriden.